### PR TITLE
RELATED: RAIL-2833 add support for drillDown to DashboardView

### PR DESCRIPTION
**Pull request by no23reason**
This downloads relevant parts of catalog and matches
any drill targets reported by insights with attributes with drillDown set.

If no onDrill is specified, the catalog load is skipped as the data is not needed.

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [x] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1943/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3570/ : SUCCESS in 14m 55s

**Comment by kandl** [comment]
ok to test

File: libs/sdk-ui-ext/src/internal/dashboardEmbedding/DashboardView/DashboardView.tsx [hunk 1]
```diff
@@ -100,20 +123,22 @@ export const DashboardView: React.FC<IDashboardViewProps> = ({
...
-                        filters={filters}
-                        filterContext={dashboardData.filterContext}
-                        onDrill={onDrill}
-                        drillableItems={drillableItems}
-                        ErrorComponent={ErrorComponent}
-                        LoadingComponent={LoadingComponent}
-                        onDashboardLoaded={handleDashboardLoaded}
-                        className="gd-dashboards-root"
-                    />
+                    <AttributesWithDrillDownProvider attributes={drillDownAttributes}>
```
[the remark this answers is not in the document]
Comment (no23reason) [reply libs/sdk-ui-ext/src/internal/dashboardEmbedding/DashboardView/DashboardView.tsx:126]: The contexts are not shared, each provider has its own value.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1946/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3574/ : SUCCESS in 10m 53s

**Review by lupko — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/627/ : SUCCESS in 36s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/888/ : SUCCESS in 21s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3581/ : SUCCESS in 14m 28s

**Pull request merged**